com.woltlab.wcf/templates/shared_radioButtonFormField.tpl

```diff
@@ -1,3 +1,16 @@
+{if !$field->isRequired()}
+	<label>
+		<input {*
+			*}type="radio" {*
+			*}name="{$field->getPrefixedId()}" {*
+			*}value="" {*
+			*}{if !$field->getFieldClasses()|empty} class="{implode from=$field->getFieldClasses() item='class' glue=' '}{$class}{/implode}"{/if}{*
+			*}{if $field->getValue() !== null && $field->getValue() == ''} checked{/if}{*
```

Comment:
Right now the `null` value is not considered in any case, shouldn’t the code look like this instead? (I haven’t tested it, please verify first!)

```suggestion
			*}{if $field->getValue() === null || $field->getValue() === ''} checked{/if}{*
```